config/jobs/kubernetes/community/community-presubmit.yaml

```diff
@@ -4,18 +4,13 @@ presubmits:
     branches:
     - master
     always_run: true
-    labels:
-      preset-service-account: "true"
+    decorate: true
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
-        args:
-        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-        - "--root=/go/src"
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - --scenario=execute
-        - --
+      - image: golang:1.11
+        command:
```

Comment:
Unfortunately I think the previous command was necessary.  For Reasons™, [podutils decoration messes with GOPATH](https://github.com/kubernetes/test-infra/blob/895df89b7e4238125063157842c191dac6f7e58f/prow/pod-utils/decorate/podspec.go#L474), so while `/go/bin` is on PATH, `/home/prow/go/bin` is not. We may be able to get rid of this in our brave new go module world, but at present I think there are some jobs that depend on this